Content guidelines.md

```diff
@@ -0,0 +1,65 @@
+# Content guidelines
+
+We know that your platform has its own voice and tone, and it’s ok to preserve them in the channel. However, channels are part of Shopify and merchants don’t see them as a separate experience.
+
+When you’re creating content for a sales channel, use Shopify’s [voice and tone guide](https://polaris.shopify.com/content/voice-and-tone) to validate your decisions.
+
+> Note
+>
+> > When building the channel, use the names of features you have on your platform without altering, so that merchants can recognize them when they leave Shopify admin and use your platform. 
+
+---
+
+## Specific content use cases
+
+Channels are a bridge between Shopify and your platform. To communicate correctly, you might require unique messaging and patterns that will differ from what merchants are used to seeing in Shopify. That’s okay.
+
+---
+
+## Communicating external errors
+
+When merchants run into errors that are caused by something out of Shopify’s control, this is how you can communicate it.
+
+|                                                                                               |                                                                                                     |
```

Comment:
Do we need Do/Don't  here? The alt text won't be visible on the image, it's just for screen readers I think